### PR TITLE
Piping person 1 into person 2 for Household schemas (ENG/WLS/NI)

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Updated the Household schema for ENG/WLS/NI so that answers for the first person in the list are piped into the answer options for any additional people that are added (`past-usual-household-address` and `usual-household-address` questions).

### How to review

Check that the  `past-usual-household-address` and `usual-household-address` questions display an additional answer option for the second person in the list onwards.

Check that the changes match the trello card.

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/piping-person-one/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/piping-person-one/schemas/en/census_individual_gb_eng.json)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/piping-person-one/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/piping-person-one/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

**Comment by berroar** [comment]
[[K] Add piping person 1 to person 2 on for relevant questions. (S)](https://trello.com/c/u8yr2G9D/3858-k-add-piping-person-1-to-person-2-on-for-relevant-questions-s)

File: source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet [hunk 1]
```diff
@@ -53,12 +53,12 @@ local proxyDescription = 'If they had no usual address one year ago, state the a
 local additionalAnswerOption = [
   {
     label: {
-      text: 'Same address as {first_person}',
+      text: 'Same as {first_person} address one year ago',
```
Comment (iwootten) [inline source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet:56]: Missing the `'s`

File: source/jsonnet/northern-ireland/household/blocks/individual/past_usual_household_address.jsonnet [hunk 2]
```diff
@@ -54,12 +54,12 @@ local proxyDescription = 'If they had no usual address one year ago, state the a
 local additionalAnswerOption = [
   {
     label: {
-      text: 'Same address as {first_person}',
+      text: 'Same as {first_person} address one year ago',
```
Comment (iwootten) [inline source/jsonnet/northern-ireland/household/blocks/individual/past_usual_household_address.jsonnet:57]: Missing the `'s`

File: source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet (hunk 1 above)
Comment (berroar) [reply source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet]: Changed the placeholder name so it is more clear now

**Pull request merged**